### PR TITLE
Fix BSON Object ID errors for CI

### PR DESCRIPTION
**Pull request by superkhau**
@0candy PTAL

### CODE REVIEW

File: test/include.test.js [hunk 1]
```diff
@@ -478,18 +487,18 @@ describe('include', function() {
...
+
+                          patients.should.have.length(1);
+                          var p = patients[0];
+                          p.name.should.equal('a');
+                          p.addressId.should.eql(patient.addressId);
+                          p.address().id.should.eql(address.id);
+                          p.address().name.should.equal('z');
+
+                          done();
+                        });
```
Comment (0candy) [inline test/include.test.js]: Does the linter fail if we revert the extra indent?

Comment (superkhau) [inline test/include.test.js]: Yes it does

Comment (superkhau) [inline test/include.test.js]: We can address this as part of homework test refactor. This super nested test needs cleanup anyways.

**Comment by 0candy** [comment]
LGTM

**Pull request merged**